storybook/stories/transactions/index.js

```diff
@@ -0,0 +1 @@
+import './TransactionsList.stories';
```

Comment:
Maybe these items could be subitems of Wallets?

Reply:
`Transactions` are their own domain within tests, the api, and now in stories. There is quite a lot that can fall under `Transactions` so I figured it needs its own domain.